src/components/Carousel.js

```diff
@@ -579,7 +580,7 @@ class Carousel extends Component {
         // if 3d is available, let's take advantage of the performance of transform
         const transformProp = CSSTranslate(currentPosition + '%', this.props.axis);
 
-        const transitionTime = this.props.transitionTime + 'ms';
+        const transitionTime = this.props.customTransition ? '0ms' : this.props.transitionTime + 'ms';
```

Comment:
Instead of setting the time of this transition to 0ms, would be better to remove it completely when a custom transition is used. 

Also, would be nice to create a story with more examples.

Reply:
Do you mean remove transition time?  There is a base transition duration in  CSS:
https://github.com/leandrowd/react-responsive-carousel/blob/d3ccfb0fbba43ca13ceb36375e192d81e61cb0f6/src/components/_carousel.scss#L184
so if not set transition time to 0, the slider would still have old "sliding" effect.

Reply:
I meant you could remove the "sliding" effect. It's set right above this on line 580.

Reply:
The sliding effect is for *slider*, and custom animation is for *slide*. We need the slider translate to position slide.

Reply:
There is an `.animated` class we use for the sliding transition. What I think would make sense here is to rename that to something like `.animate-sliding` and only use that class if there are not custom transitions. Does it make sense?

Reply:
Makes perfect sense! Updated as suggested.